webcompat/static/js/lib/bugform.js

```diff
@@ -159,10 +180,11 @@ function BugForm() {
       $("[value=" + problemType[1] + "]").click();
     }
 
-    // If we got a details param, add that to the end of the issue description.
+    // If we got a details param, add that to the end of the steps to reproduce field
+    // github.com/webcompat/webcompat.com/issues/1547#issuecomment-305649762
```

Comment:
I don't understand what the point of this comment is (specifically, the link to the GitHub issue comment). Can we be more descriptive? Or remove it if it's unintentional.